src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
Mainly because we are discouraging new use of this test infrastructure. Whomever had it working before, they should continue to use it as is. If we remove the StandardLoad call, then it might break in non-trivial ways, assuming they changed the configuration slightly from the default.